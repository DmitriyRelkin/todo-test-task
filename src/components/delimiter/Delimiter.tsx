import React from "react";
import {View} from "react-native";
import {getDelimiterStyles} from "./styles";

const Delimiter: React.FC = () => {
  const { container } = getDelimiterStyles();

  return (
    <View style={container} />
  );
};

export default Delimiter;
