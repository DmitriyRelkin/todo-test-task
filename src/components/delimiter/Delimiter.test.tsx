import React from "react";
import renderer from "react-test-renderer";
import Delimiter from "./Delimiter";


describe("Delimiter", () => {
  it("Should render component", () => {
    // Render the Delimiter component
    const tree = renderer.create(<Delimiter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
