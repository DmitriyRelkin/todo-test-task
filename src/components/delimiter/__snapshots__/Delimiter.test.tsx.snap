// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delimiter Should render component 1`] = `
<View
  style={
    {
      "borderBottomColor": "#86a5aa",
      "borderBottomWidth": 2,
      "marginBottom": 5,
      "paddingTop": 15,
      "width": "100%",
    }
  }
/>
`;
